topics/user_interface_components/status_bar_widgets.md

```diff
@@ -0,0 +1,124 @@
+[//]: # (title: Status Bar Widgets)
+
+<!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+The status bar widget implementation is used to add new widgets to the
+[status bar](https://www.jetbrains.com/help/idea/guided-tour-around-the-user-interface.html#status-bar).
+
+Widgets on the status bar are small interface elements that allow you to provide the user important information about the current file or the project.
```

Comment:
```suggestion
Status bar widgets are small UI elements that allow providing users with useful information and settings for the current file, project, IDE, etc.
```

Not sure my version sounds good, but the point is that it provides stuff not only for files and projects.

Reply:
Sounds pretty good! Thank you, fixed.